analysis.R

```diff
@@ -0,0 +1,103 @@
+#Analysis Script for R Project
+#Max Zupfer, Roman Fresquez, Rey Ortiz Bautista
+#Introduction to Biocomputing
+
+
+#Set the working directory on personal computer
+setwd("/Users/maxwellzupfer/Desktop/Biocomputing/Rproject")
+
+#Load Plots
+library(ggplot2)
+library(cowplot)
+
+#Source the supportingFunctions.R script
+source('supportingFunctions.R')
+
+#Change all files from .txt to .csv for country, use function from source: supportingFunctions.R
+TxtToCSV(directory)
+#To run on personal computer:
+TxtToCsv("/Users/maxwellzupfer/Desktop/biocomputing/Rproject/countryY/")
+
+
+#Make the Large CSV for both countries 1 and 2, use function from source: supportingFunctions.R
+#This code allows the user to decide what to do with NA's
+Combined_CSV(directory1, directory2, directory3)
+#To run on personal computer:
+Combined_CSV("/Users/maxwellzupfer/Desktop/Biocomputing/Rproject/countryX", "/Users/maxwellzupfer/Desktop/Biocomputing/Rproject/countryY", "/Users/maxwellzupfer/Desktop/Biocomputing/Rproject")
```

Comment:
You can assign "/Users/maxwellzupfer/Desktop/Biocomputing/Rproject/" as a variable and use the paste function to change the directory